rpc/rpc-wrappers.go

```diff
@@ -670,6 +671,26 @@ func (pub PublisherClient) SubmitToCT(ctx context.Context, der []byte) (err erro
 	return
 }
 
```

Comment:
This is missing the server-side RPC handler which should be defined in [`NewPublisherServer`](https://github.com/letsencrypt/boulder/blob/a71931ae8963488d947fe79660e3b0aca3dfee5e/rpc/rpc-wrappers.go#L648).